pwa-devdocs/src/_drafts/page-builder/how-to-add-your-content-type.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How to add your Page Builder content type to PWA Studio
+---
+
+We utilise a special [master format parser](master-format-parser.md) within our Page Builder integration to easily decompose data within the master format for use in a PWA Studio React component.
+
+## Prerequisites 
+
+- Have PWA Studio setup and running in your development environment.
+- Have a working Page Builder component, with the master format implemented.
+- Be able to render and visualise your content type on the existing Luma store front.
+
+## Structure
+As we require some additional steps compared to a normal we have some additional files associated with each content type, in example our `Text` content types component structure looks like the following:
+
+- `packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Text`
+  - `__tests__`
+  - `configAggregator.js`
+  - `index.js`
+  - `text.css`
+  - `text.js`
+
+`__tests__` & `index.js` are just part of PWA Studio, and won't be discussed here.
+
+### `configAggreagtor.js`
+The config aggregator provides an interface which accepts the HTML master format for a specific content type, decomposes it and returns an flat object of the properties associated with the content type.
```

Comment:
a flat object

Reply:
This will be resolved as part of: https://jira.corp.magento.com/browse/PB-35